scylla-server-rust/src/controllers/send_config_controller.rs

```diff
@@ -0,0 +1,51 @@
+use std::sync::Arc;
+
+use axum::{extract::Path, Extension};
+use axum_extra::extract::Query;
+use protobuf::Message;
+use rumqttc::v5::AsyncClient;
+use serde::Deserialize;
+use tracing::{info, warn};
+
+use crate::{command_data::CommandData, error::ScyllaError};
+
+pub const CALYPSO_BIDIR_CMD_PREFIX: &str = "Calypso/Bidir/Command/";
+
+#[derive(Deserialize, Debug)]
+pub struct ConfigRequest {
+    pub data: Vec<f32>,
+}
+
+pub async fn send_config(
```

Comment:
can you add a detailed comment to this/ throw some comments throughout this method, since it's probably gonna be used for a bunch of different commands/ it's not immediately obvious to me what all the code is doing. 

Also, IMO, we should just make different endpoints for each of the different configs that someone would want to set. That way, we'll have a lot more control over what range of values are acceptable and maybe even some kind of permissions system if we want to gate some configs behind a password or smthn